### PR TITLE
feat: add variable checks

### PR DESCRIPTION
**Pull request by benney-au-le**

### CODE REVIEW

File: scripts/db-create.sh [hunk 1]
```diff
@@ -16,6 +18,13 @@ if [ -z "$app" ]
     exit 1
 fi
 
+if [ -z "$db_container" ]
+  then
+    echo -e "${RED}Missing DB_CONTAINER env variable${NO_COLOR}"
+    exit 1
```
Comment (richardbiddle) [inline scripts/db-create.sh:24]: Please include `Usage:`

File: scripts/db-create.sh [hunk 2]
```diff
@@ -1,5 +1,7 @@
 #! /bin/bash
 set -e
+set -o errexit   # abort on nonzero exitstatus
+set -o pipefail  # don't hide errors within pipes
 
 app=${1:-$APP_NAME}
 db_container=${3:-$DB_CONTAINER}
```
Comment (richardbiddle) [inline scripts/db-create.sh:7]: This doesn't look right. what is $2?

Comment (benney-au-le) [reply scripts/db-create.sh:7]: I don't know. That's unrelated to this PR.
Can I fix that separately to this PR.

File: scripts/db-create.sh (hunk 1 above)
Comment (benney-au-le) [reply scripts/db-create.sh:25]: Added usage.

File: scripts/db-create.sh (hunk 2 above)
Comment (benney-au-le) [reply scripts/db-create.sh]: Fix argument params.

**Review by richardbiddle — COMMENTED** [review]
Can you fix the $2 and add usgae for all where appropriate please?

**Review by richardbiddle — APPROVED** [review]

**Pull request merged**